test-gensalt.c

```diff
@@ -98,6 +98,14 @@ static const char *const bcrypt_y_expected_output[] =
   "$2y$05$mAyzaIeJu41dWUkxEbn8hO"
 };
 #endif
+#if INCLUDE_yescrypt
```

Comment:
What about testing gensalt for scrypt `$7$` here as well?

Reply:
yescrypt code does not generate settings for (obsolete?) `$7$` prefix. Only crypt could encode already existing `$7$` hashes.